packages/cli/src/dbt/schema.ts

```diff
@@ -35,3 +35,54 @@ export const loadYamlSchema = async (path: string): Promise<YamlSchema> => {
     }
     return schemaFile;
 };
+
+type FindModelInYamlArgs = {
+    filename: string;
+    modelName: string;
+};
+const findModelInYaml = async ({
+    filename,
+    modelName,
+}: FindModelInYamlArgs) => {
+    try {
+        const existingYaml = await loadYamlSchema(filename);
+        const models = existingYaml.models || [];
+        const modelIndex = models.findIndex(
```

Comment:
why returning the index instead of the model with `find` ? The function name tells me this should return the whole model

Reply:
We use it later to re-insert the model at the original index in the yaml file, is there a better way?